docs/Settings.md

```diff
@@ -216,7 +216,7 @@ These parameters are access by calling [GetParam](./api/GetParam.md) or [SetPara
 | `QUIC_PARAM_STREAM_ID`<br> 0                      | QUIC_UINT62       | Get-only  | Must be called on a stream after [StreamStart](./api/StreamStart.md) is called.      |
 | `QUIC_PARAM_STREAM_0RTT_LENGTH`<br> 1             | uint64_t          | Get-only  | Length of 0-RTT data received from peer.                                              |
 | `QUIC_PARAM_STREAM_IDEAL_SEND_BUFFER_SIZE`<br> 2  | uint64_t - bytes  | Get-only  | Ideal buffer size to queue to the stream. Assumes only one stream sends steadily.     |
-| `QUIC_PARAM_STREAM_PRIORITY` <br> 3               | uint16_t          | Get/Set   | Stream priority. |
+| `QUIC_PARAM_STREAM_PRIORITY` <br> 3               | uint16_t          | Get/Set   | A value from 0x0 to 0xFFFF that indicates the Stream priority. Data on higher priority stream get sent first. All streams start with priority 0x7FFF by default.  |
```

Comment:
Looking at the bug that this fixes, I wonder if we need to be extra explicit over "higher priority == higher value".

So maybe something like "Data on streams with a higher priority value"? Or just adding another sentence with "a higher value means a higher priority"?

Up to you if you feel it is needed, but the user opening the bug looked confused with whether a high number means a high priority, or a low number means a high priority.

Reply:
I can add a line like "0xFFFF" is highest priority.